sci_analysis/test/test_graph_groupscatter.py

```diff
@@ -372,6 +372,40 @@ def test_27_scatter_two_groups_negative_corr(self):
         self.assertTrue(GraphGroupScatter(input_array['a'], input_array['b'], groups=input_array['c'],
                                           save_to='{}test_group_scatter_27'.format(self.save_path)))
 
+    def test_28_scatter_two_groups_labels(self):
+        np.random.seed(987654321)
+        input_1_x = st.norm.rvs(size=100)
+        input_1_y = [x + st.norm.rvs(0, 0.5, size=1)[0] for x in input_1_x]
+        input_2_x = st.norm.rvs(size=100)
+        input_2_y = [(x / 2) + st.norm.rvs(0, 0.2, size=1)[0] for x in input_2_x]
+        input_labels_array = np.random.choice(list('ABCDE'),  size=(200))
+        grp = [1] * 100 + [2] * 100
+        cs_x = np.concatenate((input_1_x, input_2_x))
+        cs_y = np.concatenate((input_1_y, input_2_y))
+        input_array = pd.DataFrame({'a': cs_x, 'b': cs_y, 'c': grp})
+        self.assertTrue(analyze(input_array['a'], input_array['b'], groups=input_array['c'], labels=input_labels_array,
```

Comment:
test_graph_groupscatter.py doesn't import analyze(). If you swap it out for GraphGroupScatter() I think the test will pass.